ui/eureka/zone_bozja_southern.ts

```diff
@@ -398,17 +398,18 @@ export const zoneInfoBozjaSouthern: EurekaZoneInfo = {
         fr: 'Menenius sas Lanatus',
         ja: 'メネニウス・サス・ラナトゥス',
         cn: '梅内纽斯·萨斯·拉那图斯',
-        tc: '梅內紐斯·薩斯·拉那圖斯',
         ko: '메네니우스 사스 라나투스',
+        tc: '梅內紐斯·薩斯·拉那圖斯',
       },
       shortName: {
-        en: 'Menenius', // FIX-ME
+        en: 'Menenius',
+        // FIX-ME
```

Comment:
I think these `//FIX-ME` comments here and elsewhere in this file are meant to stay with the line they are on.